dts/bindings/dma/dma.yaml

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
Should probably call this file `dma-controller.yaml` instead. See the other `*-controller.yaml` files for a template.